extensions/sql-migration/src/constants/strings.ts

```diff
@@ -233,46 +233,46 @@ export const SUMMARY_MI_TYPE = localize('sql.migration.summary.mi.type', "Azure
 export const SUMMARY_VM_TYPE = localize('sql.migration.summary.vm.type', "SQL Server on Azure Virtual Machine");
 export const SUMMARY_DATABASE_COUNT_LABEL = localize('sql.migration.summary.database.count', "Database(s) for migration");
 export const SUMMARY_AZURE_STORAGE_SUBSCRIPTION = localize('sql.migration.summary.azure.storage.subscription', "Azure storage subscription");
-export const SUMMARY_AZURE_STORAGE = localize('sql.migration.summary.azure.storage', "Azure storage");
-export const NETWORK_SHARE = localize('sql.migration.network.share', "Network Share");
-export const BLOB_CONTAINER = localize('sql.migration.blob.container.title', "Blob Container");
-export const BLOB_CONTAINER_LAST_BACKUP_FILE = localize('sql.migration.blob.container.last.backup.file.label', "Last Backup File");
+export const SUMMARY_AZURE_STORAGE = localize('sql.migration.summary.azure.storage', "Azure Storage");
+export const NETWORK_SHARE = localize('sql.migration.network.share', "Network share");
+export const BLOB_CONTAINER = localize('sql.migration.blob.container.title', "Blob container");
+export const BLOB_CONTAINER_LAST_BACKUP_FILE = localize('sql.migration.blob.container.last.backup.file.label', "Last backup file");
 export const BLOB_CONTAINER_RESOURCE_GROUP = localize('sql.migration.blob.container.label', "Blob container resource group");
 export const BLOB_CONTAINER_STORAGE_ACCOUNT = localize('sql.migration.blob.container.storage.account.label', "Blob container storage account");
-export const SOURCE_DATABASES = localize('sql.migration.source.databases', "Source Database(s)");
+export const SOURCE_DATABASES = localize('sql.migration.source.databases', "Source database(s)");
```

Comment:
We should make strings like this plural. I wonder if they localize properly

Reply:
synced with Kathy. we decided to replace all instances of (s) to the plural noun